README.md

```diff
@@ -103,11 +103,10 @@ This yielded a >20TB dataframe in Arkouda.
    - [Connecting to Arkouda](#run-ak-connect)
 5. [Logging](#log-ak)
 6. [Type Checking in Arkouda](#typecheck-ak)
-7. [Environment Variables](#env-vars-ak)
-8. [Versioning](#versioning-ak)
-9. [External Systems Integration](#external-integration)
-10. [Metrics](#metrics)
-11. [Contributing](#contrib-ak)
+7. [Versioning](#versioning-ak)
```

Comment:
I just want to check that we don't need the Environment Variables section any longer? It doesn't look like it, but want to make sure.

Reply:
@Ethan-DeBandi99 Yup, gotta be sure, and the ENVIRONMENT.md file has been gone for awhile